regexes/device/mobiles.yml

```diff
@@ -182,12 +194,121 @@ Nokia:
     - regex: 'TA-10(04|12|52)'
       model: '8'
     # 2018 Models
-    - regex: 'Nokia ([A-Za-z0-9\. ]+) Build'
-      model: '$1'
+    - regex: '(?:; )?Nokia 8\.1'
```

Comment:
All `(?:; )?` in the beginning of the following regex are useless, as they are optional

Reply:
removed